.github/workflows/canary-release.yml

```diff
@@ -0,0 +1,31 @@
+name: "Canary release"
+
+on:
+  workflow_dispatch:
+    inputs:
+      label:
+        description: "Pick a label for the release, to be included in the version number — e.g. '1.0.0-\\${label}.x' (note: can't contain slashes!)"
+        required: true
+        type: string
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    env:
+      GITHUB_TOKEN: ${{ github.token }}
+      PACKAGE_REGISTRY_ACCESS_TOKEN: ${{ secrets.NPM_TOKEN }}
+      LABEL: ${{ github.event.inputs.label }}
+    steps:
+      - uses: actions/checkout@v3
+      - uses: ./.github/actions/setup
+      - name: Create Release Pull Request or Publish to npm
+        id: changesets
+        run: |
+          npm config set "//registry.npmjs.org/:_authToken" "$NPM_TOKEN"
+      - run: |
+          git config --global user.email "cultureamp-ci@cultureamp.com"
+          git config --global user.name "cultureamp-ci"
+      - run: |
+          yarn compile
+          yarn changeset version --snapshot "$LABEL"
+          yarn changeset publish --tag "$LABEL"
```

Comment:
Now that I think about it, I think either of these even have an output, so making a comment doesn't seem possible...

Reply:
We could just have it post to slack after with the canary name if we want to track it somewhere?

Reply:
Hm, take a look at the changesets gh action, maybe instead of CLI we can use that. Cause that one outputs the versions.

I don't think having the version somewhere is overly useful if it were manually done, the value to me is in the automation. Workflow wise if I had to check for the version, copy then paste it to somewhere in slack/on this PR, then paste it to the consumer who needed it -- overall adding a step that isn't all too helpful.

Reply:
If we're going to use `changesets/action@v1` it looks like its going to be near identical with our changeset yaml? just with a label passed in the extra flags at the end for the snapshot and tag

Reply:
Should be fine, tempting to consolidate but I think people external to the team might make canary's, so I don't want all the notifications setup for package publishing to fire when a canary gets released 😅